lifecycle/lifecycle/endpoints/health.py

```diff
@@ -58,7 +58,9 @@ def is_database_connected() -> bool:
             shell(f'pg_isready -h {host} -p {port} -U {user} -d {db_name}', print_stdout=False)
 
         close_old_connections()
-        connection.cursor().execute("select 1")
+        with connection.cursor() as cursor:
+            cursor.execute('select 1')
+            cursor.fetchone()
```

Comment:
Maybe sth like `connection.close()`? I know it's possible in psycopg2 direclty, but don't know if it will work here with this django wrapper.